sql_scripts/7_beers_quarantine.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS beers_qurantine(
+
+    id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
You can only insert one field here that will reference these values in beers table through this or any other PK in beers table